workflow-dev/tpl/deis-controller-rc.yaml

```diff
@@ -53,13 +53,13 @@ spec:
                 secretKeyRef:
                   name: builder-key-auth
                   key: builder-key
-{{- if or (eq (env "DATABASE_LOCATION") "off-cluster") (eq .database_location "off-cluster") }}
```

Comment:
This previously said "if either of these says 'off-cluster'," which is not right. Preference was meant to be given to the env var, and the documentation says so.